packages/react-storage/src/components/StorageBrowser/context/actions/downloadAction.ts

```diff
@@ -0,0 +1,7 @@
+import { downloadData, DownloadDataWithPathInput } from 'aws-amplify/storage';
+
+export interface DownloadActionInput extends DownloadDataWithPathInput {}
+
+export function downloadAction(input: DownloadActionInput): void {
+  downloadData(input);
+}
```

Comment:
`downloadAction` needs to align to the `DataAction` interface, and need to see the actual values in the `input` rather than just extending from `DownloadDataWithPathInput`